server/routerlicious/packages/protocol-base/src/blobs.ts

```diff
@@ -51,6 +52,8 @@ export function getGitType(value: SummaryObject): string {
             return "commit";
         case SummaryType.Tree:
             return "tree";
+        case SummaryType.Attachment:
```

Comment:
I do not know where it is used, but should this be a "blob"? I.e. the way it is represented in storage - it's a blob